src/utils/translate/fr.json

```diff
@@ -168,6 +168,26 @@
     "Geography": "Géographie",
     "GeographyTooltip": "Sélectionner les pays concernés.",
     "Here": "Ici",
+    "IndustryMediaTitles": {
+      "UnderstandingVirus": "Comprendre le virus et ses questions sans réponse",

```

Comment:
@atatmaja @joanna-chen I feel like we should leave these titles in English. Their official title is in English and you wouldnt translate a journal article title or a book title for example into French if you were speaking about it, and you wouldn't translate the article itself if you were citing it. I think it's weird to translate because some of the translations don't literally map to french well so we'd have to basically say it in a different way in French and that would take a bit of time. Also the translation on line 178 is from Spanish so it didn't translate properly to French

Reply:
can push without translations just need to flag it here: https://docs.google.com/spreadsheets/d/1lk8ps_cKIBjFy6rVMgzHkrBJVfwKamR76bJK4ZfHmH4/edit#gid=1903680511

Reply:
> Nice work! I have 2 requests:
> 
> 1. Can we put a bit of padding between each card in the carousel
> 2. Can we add an image for each article in the media carousel, just take the first image presented in the article if applicable.

Can we do articles, reports, media for the order please? 
Would be great to have fixed heights for the cards.  
Can remove the "Type" heading labeling as article for ex. since they are segregated. 
Would put all text bellow the image, Title and a smaller date bellow that.
**Title**
Also would think about doing columns instead of carosel.